modules/RTC/JitsiLocalTrack.js

```diff
@@ -98,8 +156,10 @@ JitsiLocalTrack.prototype._setMute = function (mute) {
                         }
                     }
 
-                    if(!stream)
+                    if(!stream) {
```

Comment:
I have a concern about the call to

```
RTCUtils.obtainAudioAndVideoPermissions(streamOptions)
```
if it throws an error, you will have an uncaught error that won't neither resolve or reject your main promise. You can avoid that by adding a catch handler:

```
RTCUtils.obtainAudioAndVideoPermissions(streamOptions).then(function (streams) {
  // all the code here
}).catch(reject);
```